dashboard/app/controllers/reviewable_projects_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ReviewableProjectsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :decrypt_channel_id, only: [:create]
+
+  check_authorization
+  load_and_authorize_resource only: [:destroy]
+
+  # POST /reviewable_projects
+  def create
+    @reviewable_project = ReviewableProject.where(
+      user_id: @project_owner.id,
+      storage_app_id: @storage_app_id,
+      script_id: params[:script_id],
+      level_id: params[:level_id]
+    ).first_or_initialize
+
+    authorize! :create, @reviewable_project, @project_owner
+
+    if @reviewable_project.save
+      return render json: @reviewable_project
+    else
+      return head :bad_request
+    end
+  end
+
+  # DELETE /reviewable_projects/:id
+  def destroy
+    if @reviewable_project.delete
```

Comment:
It probably doesn't make a difference here, but do we usually use `delete` or `destroy` in code like this?

Reply:
Hmm I see `destroy` in quite a few controllers? But totally fine with changing it either way